src/Notification/Notification.less

```diff
@@ -22,10 +27,10 @@
 		&::before {
 			content: '';
 			display: block;
-			background-color: @moon-neutral-bg-color;
-			border: 6px solid @moon-black;
-			border-radius: 30px;
-			opacity: 0.85;
```

Comment:
@yeram What was the reasoning for removing this element's opacity, only to replace it with the background color and border color alpha? Borders stack on top of background. Ex: a 50% alpha border on top of a 50% alpha background results in an effective 75% total opacity for the border.